braindecode/datautil/windowers.py

```diff
@@ -154,7 +168,7 @@ def create_windows_from_events(
 def create_fixed_length_windows(
         concat_ds, start_offset_samples, stop_offset_samples,
         window_size_samples, window_stride_samples, drop_last_window,
-        mapping=None, preload=False, drop_bad_windows=True):
+        mapping=None, preload=False, drop_bad_windows=True, **epochs_kwargs):
```

Comment:
i don't like kwargs. I would expose and document here the Epochs parameters you think can be useful.

Reply:
Fixed in new commit.